nyaa/search.py

```diff
@@ -3,23 +3,115 @@
 import math
 import json
 import shlex
+from datetime import datetime, timedelta
 
 from nyaa import app, db
-from nyaa import models
+from nyaa import models, torrents
 
 import sqlalchemy_fulltext.modes as FullTextMode
 from sqlalchemy_fulltext import FullTextSearch
 from elasticsearch import Elasticsearch
 from elasticsearch_dsl import Search, Q
 
+class ElasticTorrentStatsAccessor:    
```

Comment:
`ElasticTorrentStatsAccessor` only exists to provide the `stats` property to `ElasticTorrentAccessor`.  The `stats` property is based off of `ElasticTorrentAccessor.elastic`.  With my change to `__getattr__` you could just do `ElasticTorrentAccessor.download_count` instead of `ElasticTorrentAccessor.stats.download_count`.

Now if you need to match an external interface dynamically, (not evident directly from the code with `ElasticTorrentAccessor` not being subclassed nor any comments to that effect), then that approach is fine, but should be well documented for its purpose.